frontend/src/components/Rating/Review.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { FaStar } from "react-icons/fa";
+import { useState } from "react";
+import "./Review.css";
+function Review({ mealId }) {
+  const [rating, setRating] = useState(null);
+  const [hover, setHover] = useState(null);
+  let currentRating = 1;
```

Comment:
as the rating never will be changed it should be const and also as the static value we can move it outside of the React render :) 
```
const currentRating = 1;
function Review({ mealId }) {
  const [rating, setRating] = useState(null);
```